vulnerabilities/helpers.py

```diff
@@ -292,3 +292,54 @@ def get_affected_packages_by_patched_package(
             package.vulnerable_package
         )
     return affected_packages_by_patched_package
+
+
+def get_reference_id(url: str):
+    """
+    Return the reference id from a URL
+    For example:
+    >>> get_reference_id("https://github.com/advisories/GHSA-c9hw-wf7x-jp9j")
+    'GHSA-c9hw-wf7x-jp9j'
+    """
```

Comment:
Please add an issue to improve this to support and validate many URL styles correctly (including testing the prefix is correct like in GHSA, or CVE, etc.)